packages/gestalt/src/Module.js

```diff
@@ -1,9 +1,56 @@
 // @flow strict
-import { type Node } from 'react';
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import Text from './Text.js';
+import { type BaseProps } from './ModuleTypes.js';
 import ModuleExpandable from './ModuleExpandable.js';
 
-export default function Module(): Node {
-  return null;
+export default function Module({
+  children,
+  id,
+  icon,
+  iconAccessibilityLabel,
+  title,
+  type = 'info',
+}: BaseProps): Node {
+  const MODULE_TYPE_ATTRIBUTES = {
+    info: {
+      icon,
+      color: 'darkGray',
+    },
+    error: {
+      icon: 'workflow-status-problem',
+      color: 'red',
+    },
```

Comment:
Would love to de-dup this (used here and in the Expandable version), but the use of `icon: icon` throws a kink in it. Welcome to any ideas (thinking maybe just a util function?)